src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-base-info/sw-customer-base-info.html.twig

```diff
@@ -15,6 +15,45 @@
         v-else
         class="sw-customer-base-info-columns"
     >
+        <template v-if="isBusinessAccountType && !customerEditMode">
+            <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
```

Comment:
we should not add blocks to new featues

Reply:
fixed

Reply:
Hey @keulinho 
May I ask, why there should be no new block?